pagopa/io-app
# fix(IDPay): [IODPAY-131] Add ToS and privacy links in onboarding screen

##  ⚠️ This PR depends on https://github.com/pagopa/io-app/pull/4532

## Short description
This PR adds missing ToS and Privacy links in initiative onboarding screen


https://user-images.githubusercontent.com/6160324/231734703-7b8e02e6-62a3-42e1-b5b6-8a9d69992717.mp4



## List of changes proposed in this pull request
- Added correct links to onboarding screen component

## How to test
With the `io-dev-server`, navigate to the IDPay onboarding playground and start a new onboarding. In the first screen, links in the bottom should be working
